packages/runtime/container-runtime/src/pendingStateManager.ts

```diff
@@ -101,13 +117,21 @@ function buildPendingMessageContent(
 	return JSON.stringify({ type, contents, compatDetails });
 }
 
-function withoutLocalOpMetadata(message: IPendingMessage): IPendingMessage {
+export function toStash(message: IPendingMessage): IMessageToStash {
```

Comment:
`toStash` doesn't sound as appropriate as `toSavedMessage` to describe a type of Message. `toStashMessage`, `toMessageToStash` or `toUnsavedMessage` or something may help convey the intent better